app/src/main/java/com/example/bookwormadventuresdeluxe2/BookListAdapter.java

```diff
@@ -68,21 +71,64 @@ protected void onBindViewHolder(@NonNull BookListViewHolder holder, int position
         bookListViewHolder.author.setText(book.getAuthor());
         bookListViewHolder.isbn.setText(book.getIsbn());
 
-        book.setStatusCircleColor(book.getStatus(), bookListViewHolder.statusCircle);
+        switch(this.caller) {
+            case R.id.my_books:
+                book.setStatusCircleColor(book.getStatus(), bookListViewHolder.statusCircle);
+
+                holder.itemView.setOnClickListener(new View.OnClickListener()

```

Comment:
it seems backwards to be continually setting the onclick listener here by overriding onclick. at runtime we will be continually setting the onClick listener for no reason. I think this is done elsewhere in the project so we should talk about this during the sprint.

Reply:
I take this comment back. Overriding onClick continually is necessary